indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -141,3 +140,64 @@
         padding-top: 10px;
     }
 }
+
+.submission-info {
+    .info-line {
+        margin-top: 1em;
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        .author-list {
+            font-size: 0.95em;
+            list-style-type: none;
+            margin: 0.2em 0 0 1em;
+            padding: 0;
+
+            li {
+                display: inline-block;
+                background-color: $light-gray;
+            }
+        }
+    }
+}
+
+.submitted-data {
+    .field-separator {
+        @include border-top;
+        margin: 0 -10px;
+        padding: 1em 10px;
+    }
+
+    .comment {
+        margin-bottom: 1em;
+        @extend .text-not-important;
+    }
+
+    dl.custom-field-values {
+        .field-value {
+            margin-bottom: 0.5em;
+            & > * {
```

Comment:
Empty line before the beginning of every selector inside the body of another maybe?
```scss
dl.custom-field-values {
    .field-value {
        margin-bottom: 0.5em;
     
        & > * {
            ....
        }
    }
}
```